openapi/listings/listings.yaml

```diff
@@ -123,11 +123,33 @@ components:
               description: Popularity of the source relative to other sources
               type: number
             sourceTypeCode:
-              description: 'Type of source, business directory, app (Snapchat), GPS app, etc.'
+              description: |-
+                Type of listing source
+
+                siteTypeUnspecified - The listing source type is unknown
+                siteTypeMap - Sites that provide your location on a map like MapQuest or Google Maps
+                siteTypeSocial - Social networks like Facebook
+                siteTypeAggregator - Publishers that push their listing information to other listing sources
+                siteTypeApp - Stand alone applications like Snapchat
+                siteTypeGps - These sources power GPS applications
+                siteTypeDirectory - An online listing directory
               type: string
               example: BUSINESS_DIRECTORY
             syncStatusCode:
-              description: The publishing status provided by the sources. Some sources provide confirmation that your data has been published (synced) other sources only provide confirmation that your data has been received (submitted)
+              description: |-
+                The publishing status provided by the listing sources.
+
+                statusCodeUnknown - Default error value, should never be used intentionally
+                statusCodeSynced - Listing data was submitted to the source, and the source responded that the data is good and in use
+                statusCodeInProgress - Information has been sent to the publisher who are in the process of syncing the information to the listing.
+                statusCodeSyncingWithWarnings - The data has been published by the listing but there are some warnings that many affect the values published.
+                statusCodeHasErrors - There was an error with the data we sent to the publisher causing poublishing all the information to fail.
+                statusCodeSyncDisabled - Publishing information for this source has been disabled.
+                statusCodeClaimedByOthers - Publisher is rejecting our data because this listing has been claimed by another user/service.
+                statusCodeDoesNotSupportServiceAreaBusiness - This publisher does not support service area businesses.
+                statusCodeSubmitted - The business profile information was successfully sent to the publisher. The publisher does not verify when the information was published.
+                statusCodeManualUpdateOnly - This listing cannot be updated by Vendasta's platform with the current set of apps actived for this business. The listing must be updated with the publisher directly by the business.
+                statusCodeNotFound - The publisher does not have a listing for this business.
```

Comment:
Can an AG ever have more than one listing for a source in this endpoint?

Reply:
No, 1 listing per source per AG.